xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.h

```diff
@@ -39,3 +39,14 @@ class CVideoLayerBridgeDRMPRIME
   CVideoBufferDRMPRIME* m_buffer = nullptr;
   CVideoBufferDRMPRIME* m_prev_buffer = nullptr;
 };
+
+enum drm_color_encoding {
```

Comment:
You might want to make a note about where these come from. Hopefully they are included in the public uapi at some point.

Reply:
Sure I will add a comment about origin, they are currently missing in the kernel uapi headers, I expect it will be added in the future.